src/api/v1/auth/auth.dal.ts

```diff
@@ -0,0 +1,8 @@
+import { ValidMongoId } from '../../../utils/validation.util';
+import { AuthModel } from './auth.modal';
+
+export class AuthDAL {
+  static async saveAuthTokens(userId: ValidMongoId, accessToken: string, refreshToken: string) {
+    await AuthModel.create({ userId, accessToken, refreshToken });
+  }
```

Comment:
This will create a new entry every time whenever a user logged in. Update existing record. 

Rename `saveAuthToken` to `createAuth`